package.json

```diff
@@ -7,7 +7,7 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "prettier --check 'src/server/static/**'"
```

Comment:
IMHO "test" command is not meant for linting. Let's add a different command if you want to abstract it. We should also include templates, not just static files.

Reply:
Agree, `lint` makes more sense

Reply:
I tried to define another command but it was not recognized by npm ...

Reply:
OK, didn't know about it, fine merging it as is instead of trying to debug npm

Reply:
npm run XXX worked for me

Reply:
Done in 4383116705921f0c4c3cc63023ca70e6a12fc2c1